src/editor/normalize.ts

```diff
@@ -98,13 +97,12 @@ const normalizeVoidBlock = (editor: Editor, path: Path) => {
         },
         { at: Editor.before(editor, nextPath), mode: "highest" }
       );
-      return;
     }
   }
 };
 
 /**
- * Normalizes the editor, i.e., the top-level container that holds all blocks.
+ * Normalizes the editor, i.e.,q the top-level container that holds all blocks.
```

Comment:
Q

Reply:
I swear I removed this. https://github.com/UWmathNEWS/slugline-web/pull/84/commits/a76cd9f0a23289f54e872d958c0e80885957c72f